CRM/Activity/Form/Task/PDF.php

```diff
@@ -51,7 +62,90 @@ public function postProcess() {
    * @return array
    */
   public function listTokens() {
-    return CRM_Activity_Form_Task_PDFLetterCommon::listTokens();
+    return $this->createTokenProcessor()->listTokens();
+  }
+
+  /**
+   * Create a token processor
+   *
+   * @return \Civi\Token\TokenProcessor
+   */
+  public function createTokenProcessor() {
+    return new TokenProcessor(\Civi::dispatcher(), [
+      'controller' => get_class(),
```

Comment:
get_class() here is a different class than before the patch, but it doesn't seem to make a difference on the activity print/merge. When I look around the token classes for "controller", I only really see it in hook_tokenValues. I don't see any hook implementations in universe that depend specifically on CRM_Activity_Form_Task_PDFLetterCommon.